extensions/Worker.Extensions.Http.AspNetCore/src/FunctionContext/FunctionContextExtensions.cs

```diff
@@ -47,5 +47,32 @@ internal static bool TryGetRequest(this FunctionContext context, [NotNullWhen(tr
 
             return request is not null;
         }
+
+        /// <summary>
+        /// Tries to get an HTTP Response of type T by first checking the invocation result then the output bindings of a function.
+        /// </summary>
+        /// <param name="context">The <see cref="FunctionContext"/></param>
+        /// <param name="result">The HTTP response for the respective function.</param>
+        /// <returns></returns>
+        public static bool TryGetHttpResponse<T>(this FunctionContext context, [NotNullWhen(true)] out T? result)
```

Comment:
If we are exposing a new public API, let's make sure it is properly documented. (Returns element is empty here).